rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -66,13 +66,14 @@ class Pipeline(object):
     def __init__(self, batch_size=-1, num_threads=0, device_id=-1, seed=1,
                  exec_pipelined=True, prefetch_queue_depth=2,
                  exec_async=True, bytes_per_sample=0,
-                 rocal_cpu=False, max_streams=-1, default_cuda_stream_priority=0, tensor_layout=types.NCHW, reverse_channels=False, mean=None, std=None, tensor_dtype=types.FLOAT, output_memory_type=None):
+                 rocal_cpu=False, max_streams=-1, default_cuda_stream_priority=0, tensor_layout=types.NCHW, reverse_channels=False, mean=None, std=None, tensor_dtype=types.FLOAT, output_memory_type=None, 
+                 last_batch_policy=types.LAST_BATCH_FILL, last_batch_padded=True):
```

Comment:
Make the Default as False. Its Default is False in DALI

Reply:
Should this be pad_last_batch / last_batch_padded @fiona-gladwin ?

Reply:
> Should this be pad_last_batch / last_batch_padded @fiona-gladwin ?

Reply:
Last batch padded only works for True now so we'll keep the default value as True until the issue is fixed